### ISSUE TITLE
mobile - map - square icon

### ISSUE THREAD

**Issue by IdaAkiwumi** [religious-ecologies/omeka-s-theme#57]
When the square icon is pressed on mobile it expands on the screen, but there's no way to decrease the map afterward (or go back to the previous view per say) to view the information above it and to select a different denomination. Not sure if that's been caught yet.

<img width="394" alt="Screen Shot 2020-11-11 at 4 26 14 PM" src="https://user-images.githubusercontent.com/618307/98866246-a4694e80-243a-11eb-9c1b-151a9db2a2c6.png">

**Comment by IdaAkiwumi** [comment]
I was able to minimize to previous view by sliding both sides of the page inward, but I feel there should also be an icon over the map to go back to the main view of this page when the map is expanded.

**Comment by kimisgold** [comment]
I'm going to resummarize to make sure I have this right.

On mobile resolutions, it's easy for the user to scroll down the page to view the map; however, once the map is in full view, it is difficult to navigate back up so that the instructions and filters are within frame again.

**Comment by kimisgold** [comment]
Also for clarity: the square button restores the default view of the map. So if the user pans and zooms all over the place and wants to get back to the original central location and default zoom, the square button enables resetting to that state.

**Comment by IdaAkiwumi** [comment]
Hmm, after testing this a bit more and seeing your explanation. I think what's there now should be fine.
